### ISSUE TITLE
labling the unlabled buttons in Signal for Android

### ISSUE THREAD

**Issue by pradip-sikdar** [WhisperSystems/Signal-Android#5908]
Hi.
I am Pradip Sikdar, a visually challenged android intermediate user. I have started using Signal Private Messanger for Android recently and most of the part, I am quite happy with it.
As you know, we use screen reader technology to use our android or IOS devices. So, if there is a menu or button which is not properly tagged or labled, the screen reader doesn't recognize it.
For example, let's say there is a play button. Visually there is a sign or graphics which represents that, it is a play button. But if the button is not properly tagged or labled as "Play" button, our screen reader only says "Button Unlabled". So, we cannot recognize what the button is.
So, I am here by requesting to the developer of this nice application, to lable all the buttons of this application. For example, when I am clicking on the "toggle attachment" button, I am getting some additional buttons which are not labled. So, I cannot recognize them.
Thanks in advance from all the blind community.
Pradip

**Comment by siddhartharao17** [comment]
How do I take up this bug? Like whom should I contact regarding assigning this bug to me?

I am a newbie willing to start with a simple bug.

**Comment by siddhartharao17** [comment]
@pradip-sikdar 
How do I take up this bug? Like whom should I contact regarding assigning this bug to me?

I am a newbie willing to start with a simple bug.

**Comment by pradip-sikdar** [comment]
Actually, I am also new to this platform. That's why, cannot give you any proper reference. and I think, this issue is already fixed! Thank you for your kind interest to fix this. May someone will point you to the right direction. I am just filing the accessibility bugs on behalf of the blind community.
